src/convert/traits.rs

```diff
@@ -1,6 +1,3 @@
-//! This is mostly an internal module, no stability guarantees are provided. Use
-//! at your own risk.
```

Comment:
Are we going to make this stuff part of the stable interface now?

Reply:
Nah I hope to still keep this as an internal implementation detail (these docs just moved to `src/convert/mod.rs`). I'm not sure how long we'll be able to pull this off but I figured we may as well try!